.gitignore

```diff
@@ -137,3 +137,7 @@ db4/
 osx_volname
 dist/
 *.background.tiff
+
+# Rust
+src/rusty/Cargo.lock
```

Comment:
I feel like Core is an end product and thus should commit this (as opposed to libraries). This ensures this project can't be targeted by managers of dependent crates.

Reply:
We've dropped using Cargo entirely now, so this line can probably go away.